.github/workflows/makefile.yml

```diff
@@ -2,9 +2,15 @@ name: Makefile CI
 
 on:
   push:
-    branches: [ main ]
+    branches:
+    - main
+    - release-*
```

Comment:
We use vx.x.x for product branch, like 4.1.2

Reply:
Coud we remove release* and feature*, it may be private branches.
And for v*, is there any way we can identify wildcard with format v[0-9]\\.[0-9]\\.[0-9]?

Reply:
Done.
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet